oci/utils_unix.go

```diff
@@ -28,7 +28,8 @@ import (
 	"golang.org/x/sys/unix"
 )
 
-var errNotADevice = errors.New("not a device node")
+// ErrNotADevice denotes that a file is not a valid linux device.
+var ErrNotADevice = errors.New("not a device node")
```

Comment:
ErrNotDevice

Reply:
I tried to keep it close to the existing name (and to match the runc implementation). I think err "not a device" is correct here, but let me know if you feel strongly about changing.

Reply:
"a"/"an"/"the" are usually omitted in Go.